src/gdb.ts

```diff
@@ -476,6 +476,9 @@ export class GDBDebugSession extends LoggingDebugSession {
 
         if (args.executable && !path.isAbsolute(args.executable)) {
             args.executable = path.normalize(path.join(args.cwd, args.executable));
+            if(os.platform().startsWith("win")){
```

Comment:
Why the `.startsWith`. Why not compare it to 'win32'. That is officially what it is. Also make sure you run 'npm run lint'